eng/Version.Details.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
This PR should be closed as it contains a build from the wrong channel .NET 10.0.1xx SDK instead of .NET 11.0.1xx SDK